docs/tutorials/tutorial-rollup.md

```diff
@@ -24,18 +24,18 @@ sidebar_label: Aggregate data with rollup
   -->
 
 
-Apache Druid can summarize raw data at ingestion time using a process we refer to as "rollup". Rollup is a first-level aggregation operation over a selected set of columns that reduces the size of stored data.
+Apache Druid&circledR; can summarize raw data at ingestion time using a process known as "rollup". Rollup is a first-level aggregation operation over a selected set of columns that reduces the size of stored data.
```

Comment:
Can you add a link to the actual rollup docs? Either link it from existing text, or add a new sentence ("For more information...")
https://druid.apache.org/docs/latest/multi-stage-query/concepts#rollup
https://druid.apache.org/docs/latest/ingestion/rollup

Reply:
Both links added